src/screens/internal/CompareSnapshots/AnomalyModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { AnomalyModalWrapper, AnomalyButton } from './AnomalyCompare.style';
+import { Modal, Typography } from '@material-ui/core';
+import { ProjectionsPair } from 'common/models/ProjectionsPair';
+import { Metric } from 'common/metricEnum';
+import { getAnomaliesForMetric } from './utils';
+
+export default function AnomalyModal({
```

Comment:
nit: Since this *can show* a modal, but by default is just a button that lists how many anomalies there are, I'd probably name it `AnomaliesButton` or something.